autoblocks/_impl/tracer.py

```diff
@@ -75,22 +75,23 @@ def update_properties(self, properties: Dict) -> None:
         """
         self._properties.update(properties)
 
-    def send_event(
+    def _send_event_unsafe(
```

Comment:
adding `_send_event_unsafe` so that we're try-catching this entire function within the exposed `send_event` function. previously we were only try-catching the POST request but we should probably try-catch the whole thing including the merging of properties etc